README.md

```diff
@@ -431,7 +431,7 @@ Start URI that may take users directly to the specific content in the app. Read
 Name | Type | Required | Description | Example
 --- | --- | --- | --- | ---
 url | string | yes | The URL to start | theapp://login/
-package | string | yes | The name of the package to start the URI with | 'com.mycompany'
+package | string | no | The name of the package to start the URI with | 'com.mycompany'
```

Comment:
Please also mention the minimum uia2 driver version where this argument become optional